Community-Aktivitäten/README.md

```diff
@@ -108,6 +108,20 @@ Dauer: 2h
 
 Ausstattung: Wir werden aktiv an deinem Vortrag arbeiten. Bring also etwas zum Festhalten deiner Ideen mit, in aller Regel also deinen Laptop. Wir werden per Beamer Inhalte präsentieren und an einem Flipchart oder Whiteboard wichtige Gedanken festhalten.
 
+## Mentoring Speed Dating - Finde den perfect match für deine Weiterentwicklung! (JUG Ostfalen: Steven Schwenke, Lisa Rosenberg)
```

Comment:
Danke, dass Du wieder das Speed Dating veranstaltest! :+1:

Du machst uns die Bearbeitung der einzelnen CAs einfacher, wenn Du jede CA in einen eigenen PR verpackst. :-)